.github/workflows/lighting-app.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build Lighting App
+
+on:
+    push:
+        branches:
+            - main
+    pull_request:
+        branches:
+            - main
+            - "release_*"
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        container:
+            image: ghcr.io/project-chip/chip-build-efr32:90
+
+        strategy:
+            matrix:
+                platform: [thread, wifi]
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+
+            - name: Checkout submodules & Bootstrap
+              uses: ./.github/actions/checkout-submodules-and-bootstrap
+              with:
+                  platform: silabs_docker
+
+            - name: Build Lighting-App configurations
+              uses: SiliconLabsSoftware/matter-gn-build-action@v0.0.1
+              with:
+                  json-file-path: ${{ matrix.platform == 'thread' && './.github/silabs-builds-thread.json' || './.github/silabs-builds-wifi.json' }}
+                  example-app: "lighting-app"
+                  build-script: "./scripts/examples/gn_silabs_example.sh"
```

Comment:
Where is the path to the examples and path to output to be provided for this script

Reply:
Right now the action extrapolates from the example app name. In a future revision, this can be improved